wpeverest/user-registration
# UR-572 Tweak - Notice Improvisation and added new consent notice.

### All Submissions:






### Changes proposed in this Pull Request:
This PR refactor promotional notice code such as review and survey.
This PR  also introduces a notice for allow usage tracking feature. The notice will be shown after at least one day after activation.

### How to test the changes in this Pull Request:

1.Verify whether the notice is showing or not if the plugin activation and updation period is at least one day.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Notice Improvisation and added new consent notice.
